### DISCUSSION TITLE
Question Regarding min_gain_db and max_gain_db Settings in Sionna Ray Tracing Introduction

### DISCUSSION THREAD

**Discussion by yviiiii** [NVlabs/sionna#308] [Q&A]
Hello,

I was going through the Sionna Ray Tracing Introduction on your [GitHub page](https://nvlabs.github.io/sionna/examples/Sionna_Ray_Tracing_Introduction.html?highlight=min_gain_db#Site-specifc-Link-Level-Simulations), and I have a question regarding a part of the code. I noticed that `min_gain_db` and `max_gain_db` are set to -130 and 0, respectively. Is there a specific reason or basis for choosing these example values?

I'm trying to establish criteria for determining good and poor path gains in the coverage map, but I'm finding it challenging to find relevant information on this matter. Any insights or guidance would be greatly appreciated.

Thank you.

**Comment by jhoydis** [comment]
This choice is arbitrary. However, we wanted to make sure that we only pick users in locations that have some minimum coverage. If you would simply randomly draw user positions in this scene, there are good chances that a user gets no signal at all.

Also keep in mind that we assume perfect power control later on, i.e., the path loss of every user is perfectly compensated.
